apps/pie-storybook/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Storybook for PIE Design System components",
   "scripts": {
     "dev": "storybook dev -p 6006",
-    "build": "storybook build --output-dir dist",
+    "build": "storybook build --output-dir dist && cp -R ./static ./dist",
```

Comment:
This ensures to copy the animation assets stored in the `apps/pie-storybook/static` folder, so they can be deployed as well.

Reply:
I wonder if this can be done through turbo, adding a new script called something like `copy:static-assets` that depends on `build`, and then `dev` depends on the copy step

Reply:
I don't see the point, since the it's only necessary for the build step.

Reply:
That might also require adding turbo steps to the root `turbo.json` which I feel is a bit messy already. Especially if it's only relevant for this component.